testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -357,13 +341,13 @@ class MultiPlatformViewBackgroundForegroundScenario extends Scenario with _BaseP
   void _firstFrame() {
     final SceneBuilder builder = SceneBuilder();
 
-    builder.pushOffset(0, 0);
-
-    builder.pushOffset(0, 600);
+    builder.pushOffset(50, 600);
```

Comment:
Why do we need to add 50 on x?

Reply:
The screenshot will explain it better, see before/after.

Reply:
https://github.com/flutter/engine/pull/22540/files?short_path=d68b0a5#diff-d68b0a59cd0063d730bf092fc5279ee749dd4e9348710757339c80d3a68f7923